dagger.json

```diff
@@ -2,5 +2,5 @@
   "name": "chainloop",
   "sdk": "go",
   "source": "extras/dagger",
-  "engineVersion": "v0.12.2"
+  "engineVersion": "v0.11.6"
```

Comment:
this is a downgrade

Reply:
your dagger client is probably outdated, and dagger develop made that change.

Reply:
Let me try upgrade my dagger installation and see if that's the issue.

Reply:
Yep, that was the issue, I had the installation outdated.